runner/util_windows.go

```diff
@@ -29,6 +30,10 @@ func (e *Engine) startCmd(cmd string) (*exec.Cmd, io.ReadCloser, io.ReadCloser,
 	if err != nil {
 		return nil, nil, nil, err
 	}
+
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
```

Comment:
These are overwrite StdoutPipe and StderrPipe.
https://cs.opensource.google/go/go/+/refs/tags/go1.23.1:src/os/exec/exec.go;l=1067

These pipe calls above should be removed.